src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/preferences/EditPreferencesPage.java

```diff
@@ -17,21 +15,21 @@
   private WebElement emailAddressInput;
 
   private EditPreferencesPage openEditingSection(String wikiURL) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "openEditingSection" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)